documentation/docs/install-pmm/install-pmm-client/docker.md

```diff
@@ -49,15 +50,20 @@ The PMM Client Docker image is available for both x86_64 and ARM64 architectures
 
 You can now add services with [`pmm-admin`](../../use/commands/pmm-admin.md) by prefixing commands with `docker exec pmm-client`.
 
-!!! hint alert alert-success "Tips"
-    - Adjust host firewall and routing rules to allow Docker communications. ([Read more](../../troubleshoot/checklist.md)
-    - For help: `docker run --rm percona/pmm-client:3 --help`
+!!! hint alert alert-success "Tips for Docker configuration"
 
-    In the GUI:
+   - Firewall and routing rules: Ensure your host's firewall and routing rules are configured to allow Docker communications. This is crucial for Docker containers to communicate properly. For more details, see to the [troubleshooting checklist](https://chat.deepseek.com/a/troubleshoot/checklist.md).
 
-    - Select :material-view-dashboard: *PMM Dashboards* → :material-dots-circle: *System (Node)* → :material-dots-circle: *Node Overview*.
-    - In the *Node Names* menu, select the new node.
-    - Change the time range to see data.
+   - Help command: If you need assistance with PMM Client, you can run the following command to display help information: `docker run --rm percona/pmm-client:3 --help`.
+
+Steps in the UI:
+{.power-number}
+
+    1. Go to the main menu and select **Operating System (OS) > Overview**.
+
+    2. In the **Node Names** drop-down menu, choose the new node you want to monitor.
+
+    3. Modify the time range to view the relevant data for your selected node.
```

Comment:
this part seems like it's possible to add node in UI.
especially `choose the new node you want to monitor`
maybe we should rewrite it like `select new node you just registered/installed`?
And maybe replace `Steps in the UI` to `How to see monitored node in UI`, `Find your node in UI` or something like that.

Reply:
updated section